workflows/prognostic_run_diags/fv3net/artifacts/generate.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+
+def generate(bucket: str, project: str, tag: str) -> str:
+    """Generate GCS URL given inputs."""
+    for item in [bucket, project, tag]:
+        if "/" in item:
+            raise ValueError(f"Slashes not permitted in bucket, project or tag: {item}")
+    date = datetime.datetime.now().strftime("%Y-%m-%d")
+    return f"gs://{bucket}/{project}/{date}/{tag}"
+
+
+def main(args):
+    url = generate(args.bucket, args.project, args.tag)
+    print(url)
+
+
+def register_parser(subparsers):
+    parser = subparsers.add_parser(
+        "generate", help="Generate a URL from provided bucket, project, and tag."
```

Comment:
To me "generate" implies creation of a persistent resource, which is not what this CLI really does even though it does support this task. Should we stick to the existing convention "resolve-url"?

Reply:
Yeah that sounds good.